autocertkit/ssh.py

```diff
@@ -164,9 +164,8 @@ def __init__(self,
                  timeout=300,
                  password=None,
                  nowarn=False,
-                 newlineok=False,
```

Comment:
why we remove this line?

Reply:
After refactoring, the stderr is able to return, so needless to check how many newline in command, indicates use can run any command, this lib don't care what they are, even incorrect commands.